themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties

```diff
@@ -0,0 +1,224 @@
+registerWithTitle=Registeer met {0}
+registerWithTitleHtml={0}
+loginTitle=Log in naar {0}
+loginTitleHtml={0}
+impersonateTitle={0} Belichaam gebruiker
+impersonateTitleHtml=<strong>{0}</strong> Belichaam gebruiker</strong>
+realmChoice=Realm
+unknownUser=Onbekende gebruiker
+loginTotpTitle=Mobile Authenticator Setup
+loginProfileTitle=Update account informatie
+loginTimeout=U bent te lang bezig geweest met inloggen. Het loginproces begint overnieuw.
+oauthGrantTitle=Verleen Toegang
+oauthGrantTitleHtml={0}
+errorTitle=Er is een fout opgetreden...
+errorTitleHtml=Er is een fout opgetreden...
+emailVerifyTitle=Emailadres verificatie
+emailForgotTitle=Wachtwoord vergeten?
+updatePasswordTitle=Wachtwoord updaten
+codeSuccessTitle=Succescode
+codeErrorTitle=Foutcode: {0}
+
+termsTitle=Voorwaarden
+termsTitleHtml=Voorwaarden
+termsText=<p>Gedefinieerde voorwaarden</p>
+recaptchaFailed=Ongeldige Recaptcha
+recaptchaNotConfigured=Recaptcha is verplicht, maar niet geconfigureerd
+consentDenied=Toestemming geweigerd.
+
+noAccount=Nieuwe gebruiker?
+username=Gebruikersnaam
+usernameOrEmail=Gebruikersnaam of e-mail
+firstName=Voornaam
+givenName=Voornaam
+lastName=Achternaam
+familyName=Familienaam
+email=E-mailadres
+password=Wachtwoord
+passwordConfirm=Bevestig wachtwoord
+passwordNew=Nieuw wachtwoord
+passwordNewConfirm=Bevestiging nieuwe wachtwoord
+rememberMe=Ingelogd blijven
+authenticatorCode=One-time code
+address=Adres
+postal_code=Postcode
+country=Land
+emailVerified=E-mail geverifieerd
+gssDelegationCredential=GSS delegatie Credential
+loginTotpStep1=Installeer <a href="https://fedorahosted.org/freeotp/" target="_blank">FreeOTP</a> of Google Authenticator op uw mobiele telefoon. Beide applicaties zijn beschikbaar in de <a href="https://play.google.com">Google Play</a> en Apple App Store.
+loginTotpStep2=Open de applicatie en scan de barcode of voer de sleutel in
+loginTotpStep3=Voer de eenmalige code die door de applicatie is aangeleverd in en klik op 'Verzenden' om de setup te voltooien
+loginTotpOneTime=Eenmalige code
+
+oauthGrantRequest=Wilt u deze toegangsrechten verlenen?
+inResource=in
+
+emailVerifyInstruction1=Een e-mail met instructies om uw e-mailadres te verifiÃ«ren is zojuist verzonden.
+emailVerifyInstruction2=Heeft u geen verificatiecode ontvangen in uw e-mail?
+emailVerifyInstruction3=om opnieuw een e-mail te versturen.
+
+emailLinkIdpTitle=Link {0}
+emailLinkIdp1=Er is een e-mail met instructies verzonden om {0} account {1} te koppelen met uw {2} account.
+emailLinkIdp2=Heeft u geen verificatiecode ontvangen in uw e-mail?
+emailLinkIdp3=om opnieuw een e-mail te versturen.
+
+backToLogin=&laquo; Terug naar Inloggen
+
+emailInstruction=Voer uw gebruikersnaam of e-mailadres in en wij sturen u een e-mailbericht met instructies voor het aanmaken van een nieuw wachtwoord.
+
+copyCodeInstruction=Kopieer deze code en plak deze in uw applicatie:
+
+personalInfo=Persoonlijke informatie:
+role_realm-admin=Realm Beheerder
```

Comment:
Previously you used "realmbeheer". Either way the space should go, or you should go for "Realm beheren" in both cases.